docs/unified/hardkernel/recovery.md

```diff
@@ -0,0 +1,69 @@
+# Recovery
+
+## Intro
+
+The following documentation describes the process of recovering hardware from
+the brick state using an [RTE](../../transparent-validation/rte/introduction.md)
+and Dasharo open-source firmware.
+
+=== "V1000-series"
```

Comment:
V1000 for sure...

Reply:
https://github.com/Dasharo/docs/pull/903/commits/07655bc698e7aee93f2dd14e44b96ea6d1bee70f#diff-301f7a2baf0ac481e4856e6d609b3faddc82054523e990fda27bcf8d60f26cbaL9

The tab was removed altogether when transfering files to the `variants` directory.